src/SFA.DAS.Admin.Aan.Web/SFA.DAS.Admin.Aan.Web.csproj

```diff
@@ -1,38 +1,42 @@
-		<PackageReference Include="FluentValidation" Version="11.8.1" />
+		<PackageReference Include="FluentValidation" Version="11.9.0" />
 		<PackageReference Include="FluentValidation.AspNetCore" Version="11.3.0" />
-		<PackageReference Include="FluentValidation.DependencyInjectionExtensions" Version="11.8.1" />
+		<PackageReference Include="FluentValidation.DependencyInjectionExtensions" Version="11.9.0" />
 		<PackageReference Include="MediatR" Version="12.2.0" />
 		<PackageReference Include="Microsoft.ApplicationInsights.AspNetCore" Version="2.22.0" />
-		<PackageReference Include="Microsoft.Extensions.Options" Version="7.0.1" />
-		<PackageReference Include="Microsoft.Extensions.Options.ConfigurationExtensions" Version="7.0.0" />
-		<PackageReference Include="SFA.DAS.Aan.SharedUi" Version="0.1.81-prerelease-4" />
-		<PackageReference Include="SFA.DAS.AdminService.Common" Version="0.1.75" />
+		<PackageReference Include="Microsoft.Extensions.Options" Version="8.0.2" />
+		<PackageReference Include="Microsoft.Extensions.Options.ConfigurationExtensions" Version="8.0.0" />
+		<PackageReference Include="Microsoft.IdentityModel.Protocols.OpenIdConnect" Version="7.4.0" />
+		<PackageReference Include="SFA.DAS.Aan.SharedUi" Version="0.1.95-prerelease-16" />
+		<PackageReference Include="SFA.DAS.AdminService.Common" Version="0.1.134" />
 		<PackageReference Include="SFA.DAS.Configuration.AzureTableStorage" Version="3.0.84" />
-		<PackageReference Include="SFA.DAS.DfESignIn.Auth" Version="17.1.74" />
+		<PackageReference Include="SFA.DAS.DfESignIn.Auth" Version="17.1.76" />
 		<PackageReference Include="SFA.DAS.Http" Version="3.2.69" />
 		<PackageReference Include="RestEase" Version="1.6.4" />
 		<PackageReference Include="RestEase.HttpClientFactory" Version="1.6.4" />
-		<PackageReference Include="System.IdentityModel.Tokens.Jwt" Version="6.22.0" />
+		<PackageReference Include="System.IdentityModel.Tokens.Jwt" Version="7.4.0" />
 		<PackageReference Include="WebEssentials.AspNetCore.CdnTagHelpers" Version="1.0.21" />
-		<PackageReference Include="Microsoft.Extensions.Caching.StackExchangeRedis" Version="7.0.9" />
-		<PackageReference Include="Microsoft.AspNetCore.Authentication.WsFederation" Version="6.0.11" />
-		<PackageReference Include="Microsoft.AspNetCore.Mvc.Razor.RuntimeCompilation" Version="6.0.20" />
+		<PackageReference Include="Microsoft.Extensions.Caching.StackExchangeRedis" Version="8.0.2" />
+		<PackageReference Include="Microsoft.AspNetCore.Authentication.WsFederation" Version="8.0.2" />
+		<PackageReference Include="Microsoft.AspNetCore.Mvc.Razor.RuntimeCompilation" Version="8.0.2" />
 	</ItemGroup>
 
 	<ItemGroup>
 		<ProjectReference Include="..\SFA.DAS.Admin.Aan.Application\SFA.DAS.Admin.Aan.Application.csproj" />
 	</ItemGroup>
 	<ItemGroup>
+		<Content Update="appsettings.development.json">
+		  <CopyToPublishDirectory>Never</CopyToPublishDirectory>
+		</Content>
 		<Content Update="appsettings.json">
```

Comment:
you don't need this any more because appsettings.json is removed

Reply:
Done